### PR TITLE
Make task taxi park

### PR DESCRIPTION
**Pull request by alex39292**

### CODE REVIEW

File: oop/taxiPark.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+class Car {
```
Comment (SiarheiRylach) [inline oop/taxiPark.js:1]: Please add `;` everywhere where it's necessary

File: oop/taxiPark.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        this.cars = cars
+    }
+
+    set car(car) {
+        this.cars.push(car)
+    }
+
+    totalCost() {
+        let totalCost = 0
+        for (let i = 0; i < this.cars.length; i++) {
```
Comment (SiarheiRylach) [inline oop/taxiPark.js]: Please use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

File: oop/taxiPark.js (hunk 1 above)
Comment (alex39292) [reply oop/taxiPark.js:1]: add ; everywhere

File: oop/taxiPark.js (hunk 2 above)
Comment (alex39292) [reply oop/taxiPark.js]: Used reduce()

**Comment by alex39292** [comment]
Fix

**Pull request merged**